Teradata/covalent
# Feat/angular 11

## Description
Upgrade to Angular 11

### What's included?
* Updated Angular core and material to version 11

#### Test Steps
- [x] Avoid error trying to update webdriver by setting the following in your environment: 
  `export PUPPETEER_SKIP_CHROMIUM_DOWNLOAD='true'`
- [x] `npm run serve`
  Verify demos still work as expected
- [x] `npm run serve:prod`
  Verify demos still work as expected
- [x] `npm run build:lib`
  Verify builds to completion

#### General Tests for Every PR


##### Screenshots or link to StackBlitz/Plunker
